app/controllers/concerns/authorizable.rb

```diff
@@ -13,6 +13,11 @@ def ensure_authenticated
     return render_error status: :unauthorized unless current_user
   end
 
+  # PermissionsChecker service will return a true or false depending on whether the current_user's role has the provided permission_name
+  def ensure_authorized(permission_name)
+    return render_error status: :unauthorized unless ::PermissionsChecker.new(current_user, permission_name, params).call
```

Comment:
I believe we can omit the `return`.

Reply:
For consistency only, we're making our services use named not positional arguments.

Reply:
Why the explicit `::PermissionsChecker` and not `PermissionsChecker`?

Reply:
My IDE was complaining for the ::
Every method in Authorizable.rb has an explicit `return`